packages/fast-components/src/styles/patterns/button.ts

```diff
@@ -0,0 +1,235 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "../display";
+import { focusVisible } from "../focus";
+
+export const BaseButtonStyles = css`
+    ${display("inline-block")} :host {
+        font-family: var(--body-font);
+        outline: none;
+
+        --fast-button-corner-radius: calc(var(--corner-radius) * 1px);
+        --fast-button-fill-rest: var(--neutral-fill-rest);
+        --fast-button-fill-hover: var(--neutral-fill-hover);
+        --fast-button-fill-active: var(--neutral-fill-active);
+
+        --fast-button-foreground-rest: var(--neutral-foreground-rest);
```

Comment:
Are these custom properties carry-over from the initial private implementation? I thought we landed that we didn't want to get heavily involved with component-specific custom properties as an extensibility point, instead preferring selector based overrides -`fast-button.accent {}`, `fast-button::part(end) {}`